packages/flutter_test/lib/src/window.dart

```diff
@@ -81,6 +82,12 @@ class TestWindow implements ui.SingletonFlutterWindow {
     onMetricsChanged?.call();
   }
 
+  /// Just [devicePixelRatioTestValue], but automatically reset after the test finishes
```

Comment:
Nit: maybe `/// Sets the [devicePixelRatioTestValue], and automatically resets after the test finishes...`